mac/Keyman4MacIM/Keyman4MacIM/KMInputMethodAppDelegate.m

```diff
@@ -34,6 +34,19 @@
 
 NSString *const kKeymanKeyboardDownloadCompletedNotification = @"kKeymanKeyboardDownloadCompletedNotification";
 
+@implementation NSString (VersionNumbers)
+- (NSString *)shortenedVersionNumberString {
+    static NSString *const unnecessaryVersionSuffix = @".0";
```

Comment:
It would be nice if this function name were a little more clear so we don't have to study the code to find out what it means by 'shortened'. (I don't know what you all think about function name length, but I prefer to read a long function name that is more representative of exactly what the code is doing rather than be compelled to read all the code.)
Also, it would be super helpful to have a comment that shows how an example version string would be shortened.